packages/__cxstarter__/src/components/GenericTemplate/tokens/__cxstarter__GenericTemplate.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  cxGenericTemplate,
+  asGenericTemplateToken,
+} from '@bodiless/cx-templates';
+import { as } from '@bodiless/fclasses';
+import { cxSpacing } from '@bodiless/cx-elements';
+import { __cxstarter__Layout } from '../../Layout';
+import { __cxstarter__FlowContainer } from '../../FlowContainer';
+import { StyleGuide } from './Styleguide';
+
+const Default = asGenericTemplateToken({
+  ...cxGenericTemplate.Default,
+  Components: {
+    ...cxGenericTemplate.Default.Components,
+    PageWrapper: as(__cxstarter__Layout.Default),
+    // TopContent: as(__cxstarter__FlowContainer.Default),
```

Comment:
@wodenx  Gives runtime error of :

All token specifications passed to "as" must be created by a version of "asTokenSpec"

Reply:
This is another instance of [this bug](https://github.com/johnsonandjohnson/Bodiless-JS/issues/1455).  Probably esiest just to fix it in scope of this ticket,

Reply:
Also - I think we should shadow the default flow container rather than explicitly replacing it here.